assets/less/bootstrap/code.less

```diff
@@ -57,5 +57,5 @@ pre {
 // Enable scrollable blocks of code
 .pre-scrollable {
   max-height: 340px;
-  overflow-y: scroll;
+  overflow-y: auto;
 }
```

Comment:
please do not patch our dependencies, it makes upgrading them dangerous, time consuming and at a certain degree impossible.

how about changing them with a stronger selector in our code?

e.g.

```
div.pre-scrollable {

```